hooks/post_push

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

Comment:
Add explanatory comments here (in particular, mention that this is not a git hook).